js/app.js

```diff
@@ -364,19 +364,21 @@
 				this._participants.fetch();
 			}.bind(this));
 
-			this._participantsView.listenTo(this._rooms, 'change:active', function(model, active) {
-				if (active) {
-					this.setRoom(model);
-				}
-			});
+			if (this._rooms) {
+				this._participantsView.listenTo(this._rooms, 'change:active', function(model, active) {
+					if (active) {
+						this.setRoom(model);
+					}
+				});
+			}
 
 			this._sidebarView.addTab('participants', { label: t('spreed', 'Participants'), icon: 'icon-contacts-dark' }, this._participantsView);
 		},
 		/**
 		 * @param {string} token
 		 */
 		_setRoomActive: function(token) {
-			if (OC.getCurrentUser().uid) {
+			if (this._rooms) {
```

Comment:
hmm this makes the code execute for guests?!

Reply:
Sort of: `_rooms` is only created for logged in users at https://github.com/nextcloud/spreed/blob/abded2c7ba4a64b626dc0650ad426da0f33f05b5/js/app.js#L586 so the behaviour is the same as before: